src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -100,8 +115,78 @@ class AmendmentTableImpl : public AmendmentTable
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    std::set<std::pair<uint256, std::string>> toAdd;
+
+    for (auto const& _ : detail::PreEnabledAmendments)
```

Comment:
I don't use `_` in this context but Vinnie has started to.  I have sometimes used a `_` as an identifier when that identifier is not ever referenced.  A typical usage is `lock_guard`:
````
std::lock_guard<std::mutex> _(mut);  // `_` is never referenced and thus has an unimportant name
````
I do not feel strongly about this style and in the past has often caused more trouble than it is worth.  `_` *is* a legal name.  `__` is not.  And `_CaptialLetter` is not legal either.